public/components/TagEdit/TagEdit.react.js

```diff
@@ -1,186 +1,243 @@
-import React from 'react';
-import TagNameEdit from './formComponents/TagName.react';
-import TagDescriptionEdit from './formComponents/TagDescription.react';
+import React from "react";
+import TagNameEdit from "./formComponents/TagName.react";
+import TagDescriptionEdit from "./formComponents/TagDescription.react";
 
-import TagVisibility from './formComponents/TagVisibility.react';
-import SectionSelect from '../utils/SectionSelect.react';
+import TagAdBlocking from "./formComponents/TagAdBlocking.react";
+import TagVisibility from "./formComponents/TagVisibility.react";
+import SectionSelect from "../utils/SectionSelect.react";
 
-import TopicCategories from './formComponents/topic/TopicCategories.js';
-import PodcastMetadata from  './formComponents/series/PodcastMetadata.react';
-import ContributorInfoEdit from './formComponents/contributor/ContributorInfoEdit.react';
-import PublicationInfoEdit from './formComponents/publication/PublicationInfoEdit.react';
-import NewspaperBookInfoEdit from './formComponents/newspaperbook/NewspaperBookInfoEdit.react';
-import PaidContentInfoEdit from './formComponents/paidcontent/PaidContentInfoEdit.react';
-import TrackingInfoEdit from './formComponents/tracking/TrackingInformation.react.js';
-import CampaignInfoEdit from './formComponents/campaigns/CampaignInformation.react.js';
+import TopicCategories from "./formComponents/topic/TopicCategories.js";
+import PodcastMetadata from "./formComponents/series/PodcastMetadata.react";
+import ContributorInfoEdit from "./formComponents/contributor/ContributorInfoEdit.react";
+import PublicationInfoEdit from "./formComponents/publication/PublicationInfoEdit.react";
+import NewspaperBookInfoEdit from "./formComponents/newspaperbook/NewspaperBookInfoEdit.react";
+import PaidContentInfoEdit from "./formComponents/paidcontent/PaidContentInfoEdit.react";
+import TrackingInfoEdit from "./formComponents/tracking/TrackingInformation.react.js";
+import CampaignInfoEdit from "./formComponents/campaigns/CampaignInformation.react.js";
 
-
-import * as tagTypes from '../../constants/tagTypes';
+import * as tagTypes from "../../constants/tagTypes";
 
 export default class TagEdit extends React.Component {
+  constructor(props) {
+    super(props);
 
-    constructor(props) {
-        super(props);
-
-        this.renderTagTypeSpecificFields.bind(this);
-    }
-
-    onUpdateSection(e) {
+    this.renderTagTypeSpecificFields.bind(this);
+  }
 
-      const sectionId = parseInt(e.target.value, 10);
-      const section = this.props.sections.filter((section) => section.id === sectionId)[0];
+  onUpdateSection(e) {
+    const sectionId = parseInt(e.target.value, 10);
+    const section = this.props.sections.filter(
+      section => section.id === sectionId
+    )[0];
```

Comment:
could use `.find` over `.filter` here to return a single item and avoid the `[0]`

Reply:
Ah apologies, I left format on save on by accident so this file has a lot of noise in it :(